static/js/containers/CourseSearchPage.js

```diff
@@ -234,6 +236,27 @@ export class CourseSearchPage extends React.Component<Props, State> {
                 validation={error}
               />
             </div>
+            <div className="search-filters-row">
+              {facetDisplayMap.map(([name, title, labelFunction]) =>
+                (activeFacets.get(name) || []).map((facet, i) => (
```

Comment:
do we need the `|| []` check here? just wondering because in the constructor we `setState` to always have a `[]` for these values, so we shouldn't run into a case where it's `undefined` or something (as far as I can tell anyhow, haha)

Reply:
This only here to prevent flow from complaining about it, I can replace with `$FlowFixMe` instead if you prefer.

Reply:
ah, yeah I think `.get` always has a return type of `?value` so it makes sense flow would complain :/